backend/src/chats/user-chat/dto/update-user-chat.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { CreateUserChatDto } from './create-user-chat.dto';
+import { IsOptional} from 'class-validator';
+
+export class UpdateUserChatDto extends PartialType(CreateUserChatDto) {
+	//this needs to be protected for doubles
+	//since it is only one variable, @IsNotEmpty() decorator? of not?
+	@IsOptional()
+	user_id?: number;
```

Comment:
the idea was that the `@Index()` directive from the user_chats entity solves this, but since you were getting errors there we'll look at that later. if we can get that working, that should be enough, otherwise we an implement this in code in the service.

Reply:
I'll leave this comment for later as well :)